src/EdgeOrder/test/VRemove-AzEdgeOrderAddress.Tests.ps1

```diff
@@ -0,0 +1,21 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Remove-AzEdgeOrderAddress'))
```

Comment:
What is the leading V in the filename for?
Some other test files as well 
- src/EdgeOrder/test/UUInvoke-AzEdgeOrderItemCancellation.Tests.ps1
- src/EdgeOrder/test/NGet-AzEdgeOrderAddress.Tests.ps1
- src/EdgeOrder/test/VRemove-AzEdgeOrderItem.Tests.ps1